walkthroughs/howto-k8s-ingress-gateway/README.md

```diff
@@ -0,0 +1,118 @@
+# Configuring Ingress Gateway
+
+In this walkthrough, we'll configure an Ingress Gateway in our existing example color app but with a VirtualGateway resource instead of VirtualNode for ingress traffic.
+
+A virtual gateway allows resources outside your mesh to communicate to resources that are inside your mesh. The virtual gateway represents an Envoy proxy running in an Amazon ECS, in a Kubernetes service, or on an Amazon EC2 instance. Unlike a virtual node, which represents a proxy running with an application, a virtual gateway represents the proxy deployed by itself.
+
+## Prerequisites
+[Walkthrough: App Mesh with EKS](../eks/)
+
+This example requires [aws-app-mesh-controller-for-k8s](https://github.com/aws/aws-app-mesh-controller-for-k8s) version [>=v1.0.0](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md). Run the following to check the version of controller you are running.
+```
+$ kubectl get deployment -n appmesh-system appmesh-manager -o json | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'|tail -n1
+```
```

Comment:
Given this is only available in preview at the moment, shouldn't a prerequisite be setting up a preview controller?

Reply:
+1, the current link points to an older guide. I'll replace that with helm appmesh-manager link for now until we update the guide